test/__mocks__/execa.js

```diff
@@ -1,9 +0,0 @@
-module.exports = jest.fn(() =>
```

Comment:
This seems to be repeated in 3 different files. Could you please tell me what was the reason for removing it?

Reply:
Some tests relying on unmocked `execa` version which I could not find how to override in the test. Any hints?

Reply:
`jest.dontMock('execa')`?